src/toil/jobStores/googleJobStore.py

```diff
@@ -18,6 +18,18 @@
 
 GOOGLE_STORAGE = 'gs'
 
+def _retryCreateBucket(uri, headers):
```

Comment:
My reasoning was that there is nothing in the method tied to the job store so it logically doesn't belong inside the object. The import/export tests also use the method to create a non-job store bucket.